server/utils/logger.ts

```diff
@@ -131,6 +131,8 @@ export async function expressLogger(req: Request, res: Response, next: NextFunct
   req.log = log.child({
     id: req.reqId,
     user: req.user?.id,
+    timestamp: new Date().toISOString(),
+    clientIp: req.ip,
```

Comment:
This won't work as we're behind a proxy.  You'll just be logging out the address of your `nginx proxy` instance.  I imagine, in this case, you're running both the query & the nginx instance on the same IP address, so there's no difference.  Instead, you should use `req.headers['x-forwarded-for'] || req.socket.remoteAddress` I think.

There's another method, using the `trust proxy` setting in Express, which may also work.  It would allow the configuration of multiple proxies, but I don't think that's worthwhile for now.

Reply:
Yep as said thought this might be an issue! Will change. :)